src/Smp.Web/Controllers/PostsController.cs

```diff
@@ -1,43 +1,41 @@
-using System;
-using Microsoft.AspNetCore.Mvc;
-using Smp.Web.Models.Requests;
-using Smp.Web.Models;
-using Smp.Web.Services;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Authorization;
-
-namespace Smp.Web.Controllers
-{
-    [Route("api/[controller]")]
-    public class PostsController : Controller
-    {
-        private readonly IPostsService _postsService;
-        private readonly IAuthService _authService;
-
-        public PostsController(IPostsService postsService, IAuthService authService)
-        {
-            _postsService = postsService;
-            _authService = authService;
-        }
-
-        [HttpPost("[action]"), Authorize]
-        public async Task<IActionResult> CreatePost([FromBody]CreatePostRequest createPostRequest)
-        {
-            var tkn = Request.Headers["Authorization"];
-            var authorized = _authService.AuthorizeSelf(tkn, createPostRequest.SenderId) &&
-                             (_authService.AuthorizeSelf(tkn, createPostRequest.ReceiverId) ||
-                              await _authService.AuthorizeFriend(tkn, createPostRequest.ReceiverId));
-            if (!authorized) return Unauthorized();
-
-            await _postsService.CreatePost(new Post(createPostRequest));
-
-            return Ok();
-        }
-
-        [HttpGet("[action]/{userId:Guid}"), Authorize]
-        public async Task<IActionResult> GetPosts([FromRoute]Guid userId, [FromQuery]int count = 10)
-        {
-            return Ok(await _postsService.GetPostsByReceiverId(userId, count));
-        }
-    }
-}
\ No newline at end of file
+using System;
+using Microsoft.AspNetCore.Mvc;
+using Smp.Web.Models.Requests;
+using Smp.Web.Models;
+using Smp.Web.Services;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Smp.Web.Controllers
+{
+    [Route("api/[controller]"), ApiController]
+    public class PostsController : Controller
+    {
+        private readonly IPostsService _postsService;
+        private readonly IAuthService _authService;
+
+        public PostsController(IPostsService postsService, IAuthService authService)
+        {
+            _postsService = postsService;
+            _authService = authService;
+        }
+
+        [HttpPost("[action]"), Authorize]
+        public async Task<IActionResult> CreatePost([FromBody] CreatePostRequest createPostRequest)
+        {
+            var tkn = Request.Headers["Authorization"];
+            var authorized = _authService.AuthorizeSelf(tkn, createPostRequest.SenderId) &&
+                (_authService.AuthorizeSelf(tkn, createPostRequest.ReceiverId) ||
+                    await _authService.AuthorizeFriend(tkn, createPostRequest.ReceiverId));
+            if (!authorized) return Unauthorized();
+            await _postsService.CreatePost(new Post(createPostRequest));
+            return Ok();
+        }

```

Comment:
While I don't intend on trying to enforce anything, I think it's reasonable for us to use whitespace to split some things up logically.

Reply:
Addressed in 9902032.